lib/src/components/segmented-control/SegmentedControlContext.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+
+import type { SegmentedControlRootProps } from './SegmentedControlRoot'
+
+export type SegmentedControlTheme = 'primary' | 'marsh'
+
+interface SegmentedControlContextValue {
+  size: SegmentedControlRootProps['size']
+  theme: SegmentedControlTheme
+}
+
+export interface SegmentedControlProviderProps
```

Comment:
Does this need to be exported? I couldn't find it being used elsewhere.

Reply:
Good catch, no it doesn't need to be exported.

```suggestion
interface SegmentedControlProviderProps
```